guides/V1_cognitive_app.md

```diff
@@ -5,16 +5,16 @@ is: 'published'
 ---
 ## Make your App cognitive
 
-The Make it Cognitive section of your App's dashboard allows you to associate an instance (workspace) of [Watson Conversation](https://www.ibm.com/watson/developercloud/conversation.html) with your application. When you App is installed to a space, new messages will be sent to your Watson Conversation instance by Watson Work Services using the credentials you've provided in the Make it Cognitive section of the App's dashboard.
+The Make it Cognitive section of your App's dashboard allows you to associate an instance (workspace) of [Watson Assistant](https://www.ibm.com/watson/developercloud/conversation.html) with your application. When you App is installed to a space, new messages will be sent to your Watson Assistant instance by Watson Work Services using the credentials you've provided in the Make it Cognitive section of the App's dashboard.
 
-In your App's dashboard, you provide a Watson Conversation ID. This ID is found in [Watson Conversation](https://www.ibmwatsonconversation.com) by clicking View Details from the menu on your Watson Conversation Workspace.
+In your App's dashboard, you provide a Watson Assistant ID. This ID is found in [Watson Assistant](https://www.ibmwatsonconversation.com) by clicking View Details from the menu on your Watson Assistant Workspace.
```

Comment:
In your App's dashboard, you provide a Watson Assistant Workspace ID. This ID is found in [Watson Assistant](https://www.ibmwatsonconversation.com) by clicking View Details from the menu on your Watson Assistant Workspace.